src/server/qgsserver.h

```diff
@@ -124,6 +126,9 @@ class SERVER_EXPORT QgsServer
     //! Initialization must run once for all servers
     static bool sInitialised;
     static bool sCaptureOutput;
+
+    // map of QgsProject
+    QMap<QString, const QgsProject*> mProjectStore;
```

Comment:
Unless it's something completely different in scope, I'd suggest to name this `mProjectRegistry` because `registry` is the word commonly used within the rest of QGIS for holding collections of objects.